src/Sylius/Component/Core/Provider/ProductVariantMap/ProductVariantLowestPriceMapProvider.php

```diff
@@ -25,13 +26,26 @@ public function __construct(private ProductVariantPricesCalculatorInterface $cal
 
     public function provide(ProductVariantInterface $variant, array $context): array
     {
+        Assert::methodExists($this->calculator, 'calculateLowestPriceBeforeDiscount');
+
         return [
             'lowest-price-before-discount' => $this->calculator->calculateLowestPriceBeforeDiscount($variant, $context),
         ];
     }
 
     public function supports(ProductVariantInterface $variant, array $context): bool
     {
+        if (\method_exists($this->calculator, 'calculateLowestPriceBeforeDiscount')) {
```

Comment:
I think it's missing comparison negation.
```suggestion
        if (!\method_exists($this->calculator, 'calculateLowestPriceBeforeDiscount')) {
```